CHANGELOG.md

```diff
@@ -1,5 +1,56 @@
 # Release notes
 
+## v0.137
+
+#### Feat
+
+- It's now easier to create URLs for component views.
+
+    Before, you had to call `Component.as_view()` and pass that to `urlpatterns`.
+    
+    Now this can be done for you if you set `Component.Url.public` to `True`:
+
+    ```py
+    class MyComponent(Component):
+        class Url:
+            public = True
+        ...
+    ```
+
+    Then, to get the URL for the component, use `get_component_url()`:
+
+    ```py
+    from django_components import get_component_url
+
+    url = get_component_url(MyComponent)
+    ```
+
+    This way you don't have to mix your app URLs with component URLs.
+
+    Read more on [Component views and URLs](https://django-components.github.io/django-components/0.135/concepts/fundamentals/component_views_urls/).
+
+#### Deprecation
+
+- Currently, view request handlers such as `get()` and `post()` methods can be defined
```

Comment:
As mentioned here, as part of this PR I also suggest to softly depracate the use of `Component.get()` (and similar), and instead prefer to use `Component.View.get`, etc.

`Component.get()` was fine previously (although it was still just a shorthand for `Component.View.get()`). But with the extensions pattern added to the mix, I think the view-specific methods should be scoped within `Component.View`, to stick to the idea that extensions can do whatever they want within their class, but don't pollute the shared scope.